masu/external/notifications/aws/aws_notification_handler.py

```diff
@@ -26,6 +26,9 @@
 from masu.external.notifications.notification_interface import (NotificationInterface,
                                                                 NotificationInterfaceFilter)
 
+# the cn endpoints aren't supported by moto, so filter them out
+AWS_REGIONS = list(filter(lambda reg: not reg.startswith('cn-'), AWS_REGIONS))
```

Comment:
Why is this here?  if there is something that Moto can't handle, shouldn't it be in the test code?

Reply:
Ah sorry, for some reason I read this as downloader!

Reply:
The comment says exactly why it's there.

If we select any of the CN-based regions, moto doesn't sandbox the calls and the test contacts the actual AWS endpoint (and gets a `connection refused` because the endpoints don't respond outside of China), which causes test failures for the wrong reasons.

It's working around an upstream bug. I'm going to look at the moto code and see if there's an easy fix we can push upstream.

Reply:
OK, now that I see this is the notification handler, the question still stands.  By filtering out 'cn-' doesn't that effectively block our ability to handle notifications from 'cn-' regions?

I see you have already done this in the test code to get around the Moto problem, can we remove this in the production code?

Reply:
OH! OH! 

That part is definitely outside of the intended scope. I meant to only add that filter to the tests. Looks like I went too far. 

Thanks for catching it. :+1: